### ISSUE TITLE
Accuracy 

### ISSUE THREAD

**Issue by chandanadittyo** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#511]
How can I get the accuracy chart or overfitting chart from this model?
